system/lang/en.json

```diff
@@ -188,6 +189,12 @@
     "Expanse": "Expanse",
     "RotateCCW": "Rotate counter-clockwise",
     "RotateCW": "Rotate clockwise",
+    "RollDialog": {
+      "AdvancedRollOptions": "Advanced Options",
```

Comment:
worth making "advanced options" generic, or grouping it with other non-IS specific UX labels? probably has other applications in the sheet, even if they don't exist yet.

alternatively: is it possible to get at FVTT's own localization strings? 🤔

Reply:
You can, but depending on `game.i18n.localize('LIGHT.HeaderAdvanced')` seems brittle.

Also, in my experience writing i18n applications, no matter how similar strings might seem in English, when you change the context or placement of them, they sometimes will need subtly or majorly different expressions in some other language. My habit is to use a different key for every string in the UI just to be safe.